lab_1_keywords_tfidf/main.py

```diff
@@ -52,6 +52,14 @@ def calculate_frequencies(tokens: list[str]) -> Optional[dict[str, int]]:
 
 
 def get_top_n(frequencies: dict[str, Union[int, float]], top: int) -> Optional[list[str]]:
+    top = int(input("Какое количество токенов вы бы хотели получить? "))
```

Comment:
Don't use input in the laboratory works! All needed data you can find in the `start.py`.  Call functions in `start.py`